src/main/resources/migration/common/V29__add_previous_status.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE assessment ADD COLUMN previous_status varchar(64);
```

Comment:
I'm not super convinced by this.. might be simpler to go all in on using assessment event history, but we can make that decision when handle more complicated cases